impl_core/src/test/java/io/opencensus/trace/export/SpanExporterImplTest.java

```diff
@@ -158,7 +160,7 @@ public void serviceHandlerThrowsException() {
   @Test
   public void exportSpansToMultipleServices() {
     FakeServiceHandler serviceHandler2 = new FakeServiceHandler();
```

Comment:
exportNotSampledSpans() will fail. I will add more explicit tests in a followup PR.